src/GenFacades/Program.cs

```diff
@@ -495,7 +495,8 @@ private void AddTypeForward(Assembly assembly, INamedTypeDefinition seedType)
                 //       but the runtime currently throws a TypeLoadException without explicit forwarders for the nested 
                 //       types.
                 foreach (var nestedType in seedType.NestedTypes.OrderBy(t => t.Name.Value))
```

Comment:
The comment above still mentions this is done for all nested types regardless of their accessibility, etc. Would be good to update it.